.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -233,7 +233,7 @@ swap:
   joblog: job log
   jobstream: job stream
   judgement: judgment
-  '(?<!-)\bk8s\b(?!-)': Kubernetes
+  '(?<!-|\.)\bk8s\b(?!-|\.)': Kubernetes
```

Comment:
Is it not possible to set the scope to ignore code?

Reply:
Sometimes API names are not marked in code blocks, and this term is so pervasive I would be inclined to remove it entirely from the errors list. Remember, errors can cause a CI to fail, and we only want cast iron errors in this list.

In fact, I'd probably be inclined to move this to warnings or suggestions.

Reply:
WDYT @apinnick?

Reply:
I agree. Remove it or downgrade it. I think there is too great a risk of false positives for this term.

A lot of OpenShift code comes from upstream projects. Maintaining this particular 'error' could turn into a nightmare.